pkg/kubectl/describe.go

```diff
@@ -560,6 +562,25 @@ func describeReplicationController(controller *api.ReplicationController, events
 	})
 }
 
+// DaemonControllerDescriber generates information about a daemon controller
+// and the pods it has created.
+type DaemonControllerDescriber struct {
+	client.Interface
+}
+
+func (d *DaemonControllerDescriber) Describe(namespace, name string) (string, error) {
+	// TODO (Ananya): Implement Describe.
+	return describeDaemonController("I'm not a Marvel, I'm a DC!") //running, waiting, succeeded, failed)
```

Comment:
Good point, and thanks for the tip - I didn't think of that! I implemented Describe while you were commenting on the code and removed "I'm not a Marvel, I'm a DC!", let me know if the code for Describe seems reasonable.